src/app/app.component.html

```diff
@@ -12,47 +15,48 @@
         <a routerLinkActive="active" routerLink="/overview">
           <md-list-item>
             <md-icon fontSet="ncIcon" fontIcon="nc-grid-48"></md-icon>
-            <span class="menu-item">Overview</span>
+            <span class="menu-item" [ngClass]="{'hide-elements': !isPinned}">Overview</span>
```

Comment:
This is still doing it based on Javascript, now it's just adding a class.. Please do this with CSS only, by setting a class higher up...